src/common/utils/validators.py

```diff
@@ -8,16 +8,15 @@
 from enums import BuiltinDataTypes
 
 
-def valid_complex_type(valid_type: str, extended_types: List[str], get_blueprint: Callable) -> bool:
-    if valid_type == BuiltinDataTypes.OBJECT.value:
+def blueprint_extends_blueprint(
+    minimum_blueprint_type: str, blueprint_type: str, get_blueprint: Callable[..., Blueprint]
+) -> bool:
```

Comment:
I dont' really understand what this function does.

Does it check if blueprint_type extends minimum_blueprint_type?

maybe add a small docstring?